docs/Project.tex

```diff
@@ -0,0 +1,134 @@
+\documentclass{article}
+\title{Chess Editor 2.0}
+
+\begin{document}
+% Title Page
+\begin{titlepage}
+\begin{center}
+\vspace{1cm}
+\huge Chess Editor\\
+\normalsize
+\vspace{1cm}
+\today\\
+\vspace{1cm}
+\begin{tabular}{c}
+Devin M. O'Brien \\
+Preston Williamson\\
+Brandon Kyle\\
+Dakota Simpkins\\
+Tyler Wallschleger
+\end{tabular}
+\end{center}
+\end{titlepage}
+% Section 1 - Project Definition (Group Responsibility)
+\section{Project Definition}
+People like Chess and need a way to quickly develop and practice new strategies. I love
+Chess!!! Chess Editor will let people play Chess on their PC and set up the board the
+way they want to test different strategies, or just play in a different way. It will support
+playing against an AI, or playing against another person locally. It will include an account
+system to track wins and losses. For the purposes of testing, there will be an undo and
+redo button, as well as tools to set up specific board scenarios. It will be programmed in
```

Comment:
I think this to be reworded. I don't think there should be any exclamation points.